pdf/model/textencoding/encoder.go

```diff
@@ -50,20 +53,45 @@ type TextEncoder interface {
 
 // Convenience functions
 
-// doEncode converts a Go unicode string `raw` to a PDF encoded string using the encoder `enc`.
-func doEncode(enc TextEncoder, raw string) []byte {
-	encoded := []byte{}
+// doEncode1 converts a Go unicode string `raw` to a PDF encoded string using the encoder `enc`.
+// It expects that character codes will fit into a single byte.
+func doEncode1(enc TextEncoder, raw string) []byte {
```

Comment:
Do the 1 and 2 refer to the number of bytes?  Would be good to clarify that in the function name.
Would be good to have a specific name, e.g. encodeString8Bit, encodeString16Bit.

Reply:
Do we have test cases for both of these functions (do...)?

Reply:
Done. No test cases yet, but if we will make the change discussed in chat, it will be the responsibility of `x/text` package.